Resources/Prototypes/_FarHorizons/Entities/Objects/Devices/pda.yml

```
@@ -115,3 +115,130 @@
       whitelist:
         tags:
         - Write
+
+#SYNDI OPERATIVE PDA - needed for nukie pdas so they dont have sensors that show crew where they are
+
+- type: entity
+  abstract: true
+  parent: [ BaseItem, StorePresetUplink ] 
+  id: BaseSyndiOperativePDA
+  name: evil syndicate PDA (no coords...)
+  description: EVIL Personal Data Assistant... it doesnt give you cancer...
```

Since we will be having NT Nuke Ops in the future it would be good to have the information on this neutral
```suggestion
  description: Personal Data Assistant used by Nuclear Operatives... it doesn't give you cancer...
```

---

NOOO I WANTED TO BE SILLY IN MY CODE </3 yeah alright

---

I'm all for silliness haha. Just thinking ahead to when we have both. (Saving us all the hassle later)